package.json

```diff
@@ -29,17 +29,18 @@
     "backo": "^1.1.0",
     "bucketclient": "scality/bucketclient",
     "commander": "^2.11.0",
-    "eslint": "^2.4.0",
-    "eslint-config-airbnb": "^6.0.0",
-    "eslint-config-scality": "scality/Guidelines",
-    "eslint-plugin-react": "^4.2.3",
     "joi": "^10.6",
     "kafka-node": "^1.6.0",
     "node-schedule": "^1.2.0",
+    "node-zookeeper-client": "^0.2.2",
```

Comment:
Why adding `node-zookeeper-client`?

Reply:
We were requiring the module in QueuePopulator and ProvisionDispatcher class files and the linter complained we were not including the module in our `package.json`